jscomp/core/js_implementation.ml

```diff
@@ -22,6 +22,7 @@ module Ppx_entry = struct
 
   let rewrite_implementation (ast : Parsetree.structure) : Parsetree.structure =
     Ast_config.iter_on_mel_config_stru ast;
+    let ast = Melange_ffi.Utf8_string.rewrite_structure ast in
```

Comment:
Do you think this will have impact on compilation times? Is there a way to measure it?

Reply:
I don’t think so. What we were doing in the ppx before, we’re doing here now. The additional transformation is happening for Mel.inline which is a very small subset of cases